pyriemann/estimation.py

```diff
@@ -206,17 +206,21 @@ class XdawnCovariances(BaseEstimator, TransformerMixin):
     """
 
     def __init__(self, nfilter=4, applyfilters=True, classes=None,
-                 estimator='scm', xdawn_estimator='scm'):
+                 estimator='scm', xdawn_estimator='scm', baseline_cov=None):
         """Init."""
         self.applyfilters = applyfilters
         self.estimator = estimator
         self.xdawn_estimator = xdawn_estimator
         self.classes = classes
         self.nfilter = nfilter
+        self.baseline_cov = baseline_cov
 
     def fit(self, X, y):
+        # FIXME self.Xd should be init at XdawnCovariances.__init__ and be
+        # replaced as param not attribute
```

Comment:
I didn't fix this in this PR. I'm not sure whether you want to break the API or whether you think deprecation cycles are necessary.